includes/admin/class-wc-rest-stripe-payment-gateway-controller.php

```diff
@@ -86,13 +86,16 @@ public function get_payment_gateway_settings( $request = null ) {
 		try {
 			$id = $request->get_param( 'payment_gateway_id' );
 			$this->instantiate_gateway( $id );
-			return new WP_REST_Response(
+			$gateway_options = apply_filters(
+				'wc_stripe_payment_gateway_settings',
```

Comment:
Same thoughts as with the `wc_stripe_update_payment_gateway_settings` hook we're introducing. This works fine. I'm just not sold on introducing a hook if we can handle it internally as we're doing with the `enabled`, `name`, and `description` options.

Reply:
Same comment as https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2412#discussion_r950635288